okapi-core/src/main/java/org/folio/okapi/MainVerticle.java

```diff
@@ -457,7 +434,7 @@ private void checkSuperTenant(String okapiModule, Promise<Void> promise) {
       if (res.succeeded()) {
         logger.info("Deploy completed succesfully");
       } else {
-        logger.info("Deploy failed: {}", res.cause());
+        logger.info("Deploy failed", res.cause());
```

Comment:
If it is intentional that the failure is only logged and not reported via promise there should be a `//` comment saying so.

Reply:
It should probably report failure, but I'd rather not change behavior WRT this, so I'll add a comment.

Reply:
Created an issue for this: https://issues.folio.org/browse/OKAPI-816